crates/solver/src/arguments.rs

```diff
@@ -333,6 +333,11 @@ pub struct Arguments {
 
     #[clap(flatten)]
     pub score_params: score_computation::Arguments,
+
+    /// Should we skip settlements with non-positive score for solver
+    /// competition?
```

Comment:
Using `///` will cause this comment to show up in the `--help` text which doesn't seem intended here.